### PR TITLE
[BG] Add `Main_Page`

### PR DESCRIPTION
**Pull request by YDu8VchU**
Bulgarian translation for wiki main page (bg.md)

### CODE REVIEW

**Comment by YDu8VchU** [comment]
Hey, do i need to "rebase" something on my side?

**Comment by Nivalyx** [comment]
by `needs rebase` it means that your pull request contains a lot of filler commits like these :

image

((you shouldn't worry too much about it tho, as it'll be sorted out by the wiki maintainers here before the merge))

**Comment by YDu8VchU** [comment]
> by `needs rebase` it means that your pull request contains a lot of filler commits like these :
> 
> image
> 
> ((you shouldn't worry too much about it tho, as it'll be sorted out by the wiki maintainers here before the merge))

    Thank you, good to know. I will try to edit more locally in future to not overspam.

**Review by TPGPL — APPROVED** [review]

**Pull request merged**
